### PR TITLE
Fix Flake8 violations for files in the test/test_web.py

### PR DESCRIPTION
**Pull request by LucyJimenez**
xref https://github.com/airspeed-velocity/asv/issues/1042

Fix pep8 issues on `test_web.py`

Note: the fixture issues will be solved in another PR.

### CODE REVIEW

**Review by datapythonista — COMMENTED** [review]
Looks good. Can we stop excluding the file from flake8?

File: test/test_web.py [hunk 1]
```diff
@@ -23,7 +20,7 @@
     pass
 
 from . import tools
-from .tools import browser, get_with_retry, WAIT_TIME, locked_cache_dir, WIN
+from .tools import browser, get_with_retry, WAIT_TIME, locked_cache_dir, WIN  # noqa F401 '.tools.browser' imported but unused. The fixture 'browser' is necessary for several functions
```
Comment (datapythonista) [inline test/test_web.py:23]: Do you mind moving the fixture of fixtures to conftest in a separate PR before merging this, so we don't need to have this `noqa`.

CC: @dorothykiz1

**Comment by LucyJimenez** [comment]
> Looks good. Can we stop excluding the file from flake8?
@datapythonista 
Not yet. Fixture issues are still reported by flake 8.

File: test/test_web.py (hunk 1 above)
Comment (dorothykiz1) [reply test/test_web.py:23]: I worked on this

**Comment by datapythonista** [comment]
This has conflicts, can you resolve them pelase?

File: test/test_web.py [hunk 2]
```diff
@@ -174,12 +173,12 @@ def test_web_regressions(browser, basic_html):
...
             ('xpath', '//table[1]/tbody/tr[2]/td[1]'), 'params_examples.track_find_test'
-            ))
+        ))
 
         # Check that the expected links appear in the table
         regression_1 = browser.find_element_by_link_text('params_examples.track_find_test(1)')
-        regression_2 = browser.find_element_by_link_text('params_examples.track_find_test(2)')
-        bad_hash_link = browser.find_element_by_link_text(bad_commit_hash[:8])
+        regression_2 = browser.find_element_by_link_text('params_examples.track_find_test(2)')  # noqa F841 assigned to but never used, needed to check the link
+        bad_hash_link = browser.find_element_by_link_text(bad_commit_hash[:8]) # noqa F841 assigned to but never used, needed to check the link
```
Comment (datapythonista) [inline test/test_web.py:181]: Don't assign to a variable if the result is not used.

File: test/test_web.py [hunk 3]
```diff
@@ -256,8 +256,8 @@ def test_web_regressions(browser, basic_html):
         chain.move_to_element(regression_1)
         chain.perform()
 
-        popover = browser.find_element_by_css_selector('div.popover-content')
-        flotplot = browser.find_element_by_css_selector('canvas.flot-base')
+        popover = browser.find_element_by_css_selector('div.popover-content')  # noqa F841 assigned to but never used, needed to check an element
+        flotplot = browser.find_element_by_css_selector('canvas.flot-base')  # noqa F841 assigned to but never used, needed to check an element
```
Comment (datapythonista) [inline test/test_web.py:260]: Same

**Review by datapythonista — COMMENTED** [review]
Couple of comments, and you'll have to resolve the conflicts too.

**Comment by LucyJimenez** [comment]
@datapythonista  I included the last comments and resolved the conflicts. The PR is ready for review. Thanks a lot!

**Comment by datapythonista** [comment]
Thanks @LucyJimenez, very nice

**Pull request merged**